lang/c/c.py

```diff
@@ -250,16 +303,22 @@ def code_insert_is_not_null():
         actions.auto_insert(" != NULL")
 
     def code_state_if():
-        actions.insert("if () {\n}\n")
-        actions.key("up:2 left:3")
+        actions.insert("if (")
```

Comment:
I'm not sure we documented this anywhere but on the community backlog sessions we had decided to just directly use snippets here.

Reply:
Admittedly, I didn't quite know what to do with these. They existed in the original C-support, and all I did was comment out the part that breaks if you use an editor that adds closing braces/parentheses. Personally, I stopped using the "state <structure>" commands. I'm totally fine with removing them.